### PR TITLE
BASIC-2369 Renamed view & created update path

### PR DESCRIPTION
**Pull request by pookmish**
# NOT READY

# Summary
- Renamed the block to something more appropriate
- Created update path to update tables `block`, `context`, `views_display`, & any block reference fields.

# Needed By (Date)
- Soon

# Urgency
- High urgency

# Steps to Test

1. before checking out the branch, clone repo, enable the module
2. Check out the `peec-view-modes` branch of `stanford_image_styles`
3. Create a "Stanford Slide" node
4. place the "midget" block in the "Full Width Top" region to view the rendered output
5. Check out the `BASIC-2369-rename` branch of `stanford_fwb` and the `BASIC-2369` branch of `stanford_image_styles`
6. Run database updates.
7. verify block still exists in its configured location.

# Affected sites:
ds_casbs
ds_ethics
ds_fc-planning
ds_hsdo
ds_jsa-peec
ds_jumpstart-dreme
ds_jumpstart-graphicnovel
ds_jumpstart-neuro
ds_jumpstart-vptlplus
ds_lbre-dev
ds_lbre-jumpstart
ds_lbre-preview
ds_lbre-production
ds_mlk
ds_mlk-dev
ds_peec-sandbox
ds_sa-studentservices
ds_tedx
ds_undergrad
ds_undergrad-dev

sites where stanford_fwb-7.x-1.0 is not installed but in code:
ds_grand-tour
ds_heidegger
ds_religious-studies
ds_sws-build-jsa
ds_sws-uat-jsa

### CODE REVIEW

**Comment by pookmish** [comment]
Feel free to let me know if you think of other tables/configs that might need to be changed. also if you wanna move assignment to @jbickar, go for it.

**Comment by jbickar** [comment]
I'm unable to get the block to appear correctly, even before checking out this branch. I'm on `7.x-1.x`.

Does it have some dependencies on one of the JS* profiles? This is on a stock Stanford profile install.

**Comment by pookmish** [comment]
I had some trouble getting it to display as well. took some unknown cache clear or something to get it to display using context. standard block config seems to work fine though.

**Comment by sherakama** [comment]
There is a dependency on the theme, and at one point, there was a dependency on stanford_jumpstart for some css. We moved all the css in to the module to eliminate the need for stanford_jumpstart to be enabled but the theme has to be at a recent version.

I'll follow up with further review in a short while.

**Comment by jbickar** [comment]
Nitpicky, but I think we should:
1. Name it "Short" block (rather than "Shorts")
2. Change the machine name accordingly
3. Call the image style `full_width_banner_short_scale_and_crop_1600x400`

Also, what module was providing the `full_width_banner_midget_scale_and_crop_1600x400` image style? I'm not seeing that in `stanford_image_styles`

**Comment by pookmish** [comment]
I wanted to name it "short" but i figured for easier serialized array issues, keeping the character length the same reduced the amount of logic in the update. `midget` -> 6, `shorts` -> 6... thoughts?

I think the image style `full_width_banner_midget_scale_and_crop_1600x400` was supposed to be included in this module but it didnt get added.

**Comment by jbickar** [comment]
Oh, let me check some things. This may only be in use on one site, therefore an upgrade path might not be necessary.

**Comment by jbickar** [comment]
It is in the `peec-view-modes` branch of `stanford_image_styles`

File: stanford_fwb.features.inc [hunk 1]
```diff
@@ -19,3 +19,27 @@ function stanford_fwb_ctools_plugin_api($module = NULL, $api = NULL) {
 function stanford_fwb_views_api($module = NULL, $api = NULL) {
   return array("api" => "3.0");
 }
+
+/**
+ * Implements hook_image_default_styles().
+ */
+function stanford_fwb_image_default_styles() {
```
Comment (sherakama) [inline stanford_fwb.features.inc]: Where was the original style saved? stanford_image_styles?

Comment (pookmish) [reply stanford_fwb.features.inc]: > It is in the peec-view-modes branch of stanford_image_styles

**Comment by jbickar** [comment]
Yeah, we may not have to go with the upgrade path if this is only in use on one site. Hold please.

File: stanford_fwb.features.inc (hunk 1 above)
Comment (sherakama) [reply stanford_fwb.features.inc]: oh boy. So this module was dependant on an image style on a non-standard branch? yikes.

**Comment by jbickar** [comment]
I can confirm that the upgrade path works when checking out the `peec-view-modes` branch of `stanford_image_styles`. I'll take a pass at adding the image style to `stanford_image_styles` and renaming to "short" instead of "shorts".

**Comment by jbickar** [comment]
I'm refactoring to put the image style in `stanford_image_styles`. Hold on this.

**Comment by jbickar** [comment]
@sherakama, you will see from the other PR that I toyed with changing the short name of the image style but that failed. @pookmish and I discussed this and agreed that it is not worth the extra effort to unserialize the View setting, change the machine name, and re-serialize, so we're sticking with the simple string replacement and just changing the human-readable name.

**Review by jbickar — APPROVED** [review]
My comments are in the thread.

**Comment by sherakama** [comment]
I tested this on peec. Works really well. Good Job all.

**Pull request merged**